hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/realtime/HoodieRealtimeFileSplit.java

```diff
@@ -36,16 +38,20 @@
 
   private String basePath;
 
+  private Option<HoodieVirtualKeyInfo> hoodieVirtualKeyInfoOpt = Option.empty();
+
   public HoodieRealtimeFileSplit() {
     super();
   }
 
-  public HoodieRealtimeFileSplit(FileSplit baseSplit, String basePath, List<String> deltaLogPaths, String maxCommitTime)
+  public HoodieRealtimeFileSplit(FileSplit baseSplit, String basePath, List<String> deltaLogPaths, String maxCommitTime,
+                                 Option<HoodieVirtualKeyInfo> hoodieVirtualKeyInfoOpt)
```

Comment:
I think its okay to drop the `Opt` everywhere.  `virtualKeyInfo` is concise and already conveys the meaning. `virtualKeyInfo.isPresent()` will again convey what `Opt` would have conveyed. 

Also lets drop `hoodie` prefix everywhere in the variables as well.

Reply:
encoding data types in the name is no longer needed in the age of modern IDEs. says. - Robert C Martin in Clean Code :)